homeassistant/components/lg_soundbar/config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Config flow to configure the LG Soundbar integration."""
+from queue import Queue
+import socket
+
+import temescal
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT
+
+from .const import DEFAULT_PORT, DOMAIN
+
+DATA_SCHEMA = {
+    vol.Required(CONF_HOST): str,
+    vol.Required(CONF_PORT, default=DEFAULT_PORT): vol.Coerce(int),
```

Comment:
Do we need to ask the port? Can the port ever be different?

Reply:
I see. That seems like an edge case we shouldn't put in our main flow. That can be added later as an advanced option.

Reply:
Can the port ever be different? i can not confirm this, there is little documention about this provided by LG